incuna_auth/middleware/login_required.py

```diff
@@ -57,6 +57,9 @@ def __init__(self, check=True):
         if check:
             check_request_has_user()
 
+    def get_unauthorised_redirect_url(self, request):
+        return settings.LOGIN_REDIRECT_URL
```

Comment:
I don't think that's really a problem. You could call it `default_unauthorised_redirect_url` if you want.